Exercise-master/src/test/java/org/upgrad/controllers/UserControllerTest.java

```diff
@@ -188,29 +190,29 @@ public void loginWithBadCredentials() throws Exception{
                .andExpect(content().string(containsString("Invalid Credentials")));
     }
 
-//    @Test
-//    public void logoutUser() throws Exception{
-//        User user = new User();
-//        user.setUserName("upgrad");
-//        user.setRole("user");
-//        session = new MockHttpSession();
-//        session.setAttribute("currUser", user);
-//        String url = "/api/user/logout";
-//        mvc.perform(post(url).session(session))
-//                .andExpect(status().is2xxSuccessful())
-//                .andExpect(content().string(containsString("You have logged out successfully")));
-//    }
-//
-//    @Test
-//    public void logoutUserWithoutLogin() throws Exception{
-//        session = new MockHttpSession();
-//        session.setAttribute("currUser", null);
-//        String url = "/api/user/logout";
-//        mvc.perform(post(url).session(session))
-//                .andExpect(status().is4xxClientError())
-//                .andExpect(content().string(containsString("You are currently not logged in")));
-//    }
-//
+    @Test
+    public void logoutUser() throws Exception{
+        Users user = new Users();
```

Comment:
Same comment as above.  The original file has User user = new User();  Should we really change?